examples/create-react-app-with-styled-components-engine/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
To match what we do in the demos?

```suggestion
import * as React from 'react';
```